src/components/TestNav.js

```diff
@@ -1,11 +1,26 @@
 import React, { Component } from 'react';
 import Nav from './Nav/Nav';
+import TCLogin from './TestComp/TCLogin';
 
 export class TestNav extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loginModal: false,
+    };
+  }
+  toggleLogin = () => {
+    const { loginModal } = this.state;
+    this.setState({
+      loginModal: !loginModal,
+    });
+  };
   render() {
+    const { loginModal } = this.state;
     return (
-      <div>
-        <Nav />
+      <div style={{ backgroundColor: 'blue', height: '100%' }}>
+        <Nav toggle={this.toggleLogin} />
+        {loginModal ? <TCLogin toggle={this.toggleLogin} /> : null}
```

Comment:
```suggestion
        {loginModal && <TCLogin toggle={this.toggleLogin} />}
```
굳이 삼항연산자 없이 and 연산자 통해서 표현 가능해보입니다